vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -296,6 +296,39 @@ testFips() {
   echo "$test:Finish"
 }
 
+testCloudInit() {
+  test="testCloudInit"
+  echo "$test:Start"
+  os_sku=$1
+
+  # Limit this test only to Mariner or Azurelinux 
+  if [[ "$os_sku" == "CBLMariner" || "$os_sku" == "AzureLinux" ]]; then
+    FILE=/var/log/cloud-init.log
+    if test -f "$FILE"; then
+      echo "Check cloud-init log exist."
```

Comment:
Nit: The language here is a little clumsy -- it kind of implies that you're about to do the check, not that it already succeeded.

Suggestion:
Before doing the check, log something like `Checking if cloud-init log exists.`
Then, here, log something like `Cloud-init log exists`
Then, blow -- if they log doesn't exist -- log something like `Cloud-init log does not exist.`